moj-node/server/middleware/auth.js

```diff
@@ -18,6 +22,8 @@ const formErrors = {
     'There is a problem with the username or password you have entered. Check and try again',
   accountProblem:
     'There is a problem with your account and we are unable to log you in',
+  lockedOut: 'You have been locked out for 5 minutes',
```

Comment:
Could actually tie this number into the const setting at the top of the page so if you change one it changes the message automatically :)

Reply:
Yes, this